cockpit/gui/imageViewer/viewCanvas.py

```diff
@@ -813,3 +823,13 @@ def resetPixelScale(self):
         self.image.autoscale()
         self.histogram.lthresh, self.histogram.uthresh = self.image.getDisplayRange()
         self.Refresh()
+
+    def saveData(self, evt=None):
+        with wx.FileDialog(self, "Save image", wildcard="DV files (*.dv)|",
+                           style=wx.FD_SAVE | wx.FD_OVERWRITE_PROMPT) as fileDialog:
+            if fileDialog.ShowModal() == wx.ID_CANCEL:
+                return
+            else:
+                path = fileDialog.GetPath()
+        from cockpit.util.datadoc import writeDataAsMrc
```

Comment:
Any special reason why this is not a top level import?

Reply:
It's only used here, and I thought this makes it obvious where writeDataAsMrc comes from with minimal clutter. For the same clarity with a top level import, I'd rather import the module, but that means a long call string here: ```cockpit.util.datadoc.writeDataAsMrc```

Reply:
The long call is still well under 80 columns. Resolved in 2598b3c4df9f17645c8e42413dba19c66abe3b70.